### PR TITLE
SLING-10277 - REGISTER PRIVILEGE and SET ACL should support lists of …

### PR DESCRIPTION
**Pull request by bdelacretaz**
…STRING and NAMESPACED_ITEM

### CODE REVIEW

File: src/test/resources/testcases/test-10.txt [hunk 1]
```diff
@@ -4,6 +4,7 @@
 set ACL on /libs,/apps, /, /content/example.com/some-other_path
     remove * for user1,user2
     allow jcr:read for user1,user2
+    allow noNamespace for user4
```
Comment (anchela) [inline src/test/resources/testcases/test-10.txt:7]: for the sake of clarity i would probably change 'noNamespace' to 'privilege_without_namespace' to make it unambigous that this is actually a privilege name (and not a namespace thing).

File: src/test/resources/testcases/test-99.txt [hunk 2]
```diff
@@ -13,6 +13,7 @@ create service user user1, u-ser_2
 set ACL on /libs,/apps
     remove * for user1,u-ser_2
     allow jcr:read for user1,u-ser_2
+    allow noNamespace for user4
```
Comment (anchela) [inline src/test/resources/testcases/test-99.txt:16]: same here....

**Review by anchela — APPROVED** [review]
@bdelacretaz the patch looks good to me.... just had a comment regarding the privilege name 'noNamespace' which doesn't make it perfectly clear that it's actually a privilege name.

File: src/test/resources/testcases/test-10.txt (hunk 1 above)
Comment (bdelacretaz) [reply src/test/resources/testcases/test-10.txt]: Makes sense, I'll do that

**Pull request merged**